### PR TITLE
Add install-setup script for Ubuntu.

### PR DESCRIPTION
**Pull request by 130s**
DO NOT MERGE YET. Not fully tested.

### CODE REVIEW

**Comment by 130s** [comment]
Good for the review.

File: hironx_ros_bridge/robot/visionpc_install_setup.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+PKG_TO_INSTALL="$PKG_TO_INSTALL $PKG_OSS_DEV"
+
+# For ROS
+PKG_ROS="ros-hydro-rtmros-nextage ros-hydro-hironx-tutorial ros-hydro-rtshell-core"
+PKG_TO_INSTALL="$PKG_TO_INSTALL $PKG_ROS"
+
+# Setup git account for tork user just in case for git commit work.
+cd ~
+wget https://raw.githubusercontent.com/130s/compenv_ubuntu/master/dot_gitconfig && mv dot_gitconfig .gitconfig
+wget https://raw.githubusercontent.com/130s/compenv_ubuntu/master/dot_gitignore_global && mv dot_gitignore_global .gitignore_global
```
Comment (k-okada) [inline hironx_ros_bridge/robot/visionpc_install_setup.sh]: Do we need this line?

Comment (130s) [inline hironx_ros_bridge/robot/visionpc_install_setup.sh]: I see, removed. Maybe each one who does installation work adds these personal setting if necessary.

Comment (k-okada) [inline hironx_ros_bridge/robot/visionpc_install_setup.sh]: do you mean this line is required for installation work?

Comment (130s) [inline hironx_ros_bridge/robot/visionpc_install_setup.sh]: No sorry, not necessary.

**Pull request merged**